index.html

```diff
@@ -36,21 +37,21 @@ <h2>Filter Starred Ideas</h2>
         </div>
       </form>
       <!-- BOTTOM -->
-      <section class="card-holder">
-        <article class="card">
+      <section class="card-grid">
+        <!-- <article class="card hidden">
```

Comment:
I approve of the change for removing the additional article! I looks cleaner and it allows us to work more thoroughly with dev tools to pinpoint what will be needed for our future quarry selectors!